src/SoftekBarcodeAPI.php

```diff
@@ -0,0 +1,37 @@
+<?php
+declare(strict_types=1);
+
+namespace DexproSolutionsGmbh\SoftekBarcodeWrapper;
+
+use DexproSolutionsGmbh\SoftekBarcodeWrapper\Model\SoftekBarcodeConfig;
+use FFI;
+
+class SoftekBarcodeAPI
+{
+    /**
+     * The foreign C API
+     *
+     * @var FFI
+     */
+    protected FFI $interface;
```

Comment:
I'd call this `$ffi` instead, it is shorter and still explicit about the fact that this is the FFI interface.